sig/prototype/app/helpers/application_helper.rbs

```diff
@@ -9,5 +9,7 @@ module ApplicationHelper
 
   def application_url: () -> untyped
 
+  def page_title: (?::String page_title) -> untyped
```

Comment:
```suggestion
  def page_title: (?::String page_title) -> string
```

Reply:
Thank you! Changed it to `string` with https://github.com/kaigionrails/conference-app/pull/256/commits/b595e690f7a46cd3d977995e3f1845cdb9cab6d7